vm/src/stdlib/math.rs

```diff
@@ -172,6 +172,43 @@ fn math_lgamma(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn inner_rounding(func_name: &str, vm: &VirtualMachine, value: &PyObjectRef) -> PyResult {
```

Comment:
Rename this function so that it does not suggest to do rounding. Something like `call_magic_method` or `try_magic_method`.

Reply:
Thanks, fixed it